lambdas/service/app.py

```diff
@@ -1436,8 +1449,12 @@ def _dss_files(uuid, fetch=True):
                                                      Params=params)
         response = {"Status": 302, "Location": location}
     else:
-        dss_response.raise_for_status()
-        assert False
+        try:
+            dss_response.raise_for_status()
+        except requests.exceptions.HTTPError as e:
```

Comment:
What's the intent here? It appears that any non-success HTTP status code (4xx 5xx etc) from DSS is translated to a 400. That can't be right.

Reply:
`raise_for_status` raises `HTTPError` which isn't caught by chalice resulting in a stacktrace being printed instead of a JSON response showing the error message. I was under the impression we were trying to avoid that.

Reply:
I see. I'm assuming by "printed" you mean sent back to the client as part of the response. That bhaviour is controlled by AZUL_DEBUG. See AzulChaliceApp. If this is the only reason this change was made, we can revert the change.